ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/TestData.kt

```diff
@@ -278,7 +279,8 @@ internal class MockViewModel(
     }
 }
 
-internal data class TestStoreProduct(
+@VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
+data class TestStoreProduct(
```

Comment:
#1333

Reply:
Thanks! I will rebase on top of paywalls when that's merged